jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -237,9 +236,13 @@ def _on_add_data_to_viewer(self, msg):
 
                     try:
                         if angle_unit is None:
-                            # default to sr if input spectrum is not in surface brightness units
-                            # TODO: for cubeviz, should we check the cube itself?
-                            self.angle_unit.selected = 'sr'
```

Comment:
why remove this?  It doesn't seem to be replaced with any other logic to accomplish the same thing, except the `else` below which is defaulting to pix2 instead.

Reply:
The logic just gets moved, not removed. So if the `angle_unit` is `None` and the config is not specviz nor specviz2d, then the `angle_unit` will still be set to `sr`. By updating the `has_angle` and `has_sb` if it is one of the 2 configs, it removes the `sb_unit` and `angle_unit` from the UI/API.

I can reintroduce the comments that previously exists about setting the `angle_unit` to `sr`.